2021-01-10 12:57:36,575 - GPTime.model.train - INFO - Device: cuda:0
2021-01-10 12:57:36,614 - GPTime.networks.mlp - DEBUG - Input size of model: 256.
2021-01-10 12:57:36,614 - GPTime.networks.mlp - DEBUG - Number of layers: 10.
2021-01-10 12:57:36,615 - GPTime.networks.mlp - DEBUG - Number of hidden units: 512.
2021-01-10 12:57:37,850 - GPTime.model.train - INFO - Number of learnable parameters: 2233345
2021-01-10 12:57:37,855 - GPTime.model.train - INFO - Using learning rate sheduler: None
2021-01-10 12:57:37,856 - GPTime.model.train - INFO - Not using a validation set. Training on the full dataset.
2021-01-10 12:57:37,857 - GPTime.model.data - INFO - Loading dir: GPTime/data/processed/M4/dir0000
2021-01-10 12:58:08,854 - GPTime.model.data - INFO - Total of 100000 time series in memory.
2021-01-10 12:58:08,854 - GPTime.model.data - INFO - Frequencies: {'D': True, 'H': True, 'M': True, 'O': True, 'Q': True, 'W': True, 'Y': True}.
2021-01-10 12:58:08,904 - GPTime.model.train - INFO - seasonal init: False
2021-01-10 12:58:08,904 - GPTime.model.train - INFO - Training model.
2021-01-10 12:58:08,905 - GPTime.model.train - INFO - Length of dataset: 100000
2021-01-10 12:58:29,280 - GPTime.model.train - INFO - Changed learning rate. Current lr = 0.0005
2021-01-10 12:58:49,350 - GPTime.model.train - INFO - Changed learning rate. Current lr = 0.00025
2021-01-10 12:59:09,454 - GPTime.model.train - INFO - Changed learning rate. Current lr = 0.000125
2021-01-10 12:59:29,512 - GPTime.model.train - INFO - Changed learning rate. Current lr = 6.25e-05
2021-01-10 12:59:49,590 - GPTime.model.train - INFO - Changed learning rate. Current lr = 3.125e-05
2021-01-10 12:59:49,590 - GPTime.model.train - INFO - Epoch 5     [Avg. Loss, Loss]: [24.8045, 2.1968], 0
2021-01-10 13:00:09,670 - GPTime.model.train - INFO - Changed learning rate. Current lr = 1.5625e-05
2021-01-10 13:00:29,730 - GPTime.model.train - INFO - Changed learning rate. Current lr = 7.8125e-06
2021-01-10 13:00:49,787 - GPTime.model.train - INFO - Changed learning rate. Current lr = 3.90625e-06
2021-01-10 13:01:09,734 - GPTime.model.train - INFO - Changed learning rate. Current lr = 1.953125e-06
2021-01-10 13:01:29,650 - GPTime.model.train - INFO - Changed learning rate. Current lr = 9.765625e-07
2021-01-10 13:01:29,651 - GPTime.model.train - INFO - Epoch 10    [Avg. Loss, Loss]: [1.7289, 1.6590], 0
2021-01-10 13:01:49,555 - GPTime.model.train - INFO - Changed learning rate. Current lr = 4.8828125e-07
2021-01-10 13:02:09,477 - GPTime.model.train - INFO - Changed learning rate. Current lr = 2.44140625e-07
2021-01-10 13:02:29,343 - GPTime.model.train - INFO - Changed learning rate. Current lr = 1.220703125e-07
2021-01-10 13:02:49,306 - GPTime.model.train - INFO - Changed learning rate. Current lr = 6.103515625e-08
2021-01-10 13:03:09,171 - GPTime.model.train - INFO - Changed learning rate. Current lr = 3.0517578125e-08
2021-01-10 13:03:09,171 - GPTime.model.train - INFO - Epoch 15    [Avg. Loss, Loss]: [1.6401, 1.6475], 0
2021-01-10 13:03:09,172 - GPTime.run - ERROR - Task train failed
Traceback (most recent call last):
  File "/share/apps/software/Python/3.7.4-GCCcore-8.3.0/lib/python3.7/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/share/apps/software/Python/3.7.4-GCCcore-8.3.0/lib/python3.7/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/lustre1/home/erikde/master_thesis/GPTime-Series/GPTime/__main__.py", line 5, in <module>
    main_cli()
  File "/lustre1/home/erikde/master_thesis/GPTime-Series/venv/lib/python3.7/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/lustre1/home/erikde/master_thesis/GPTime-Series/venv/lib/python3.7/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/lustre1/home/erikde/master_thesis/GPTime-Series/venv/lib/python3.7/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/lustre1/home/erikde/master_thesis/GPTime-Series/venv/lib/python3.7/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/lustre1/home/erikde/master_thesis/GPTime-Series/GPTime/run.py", line 50, in main_cli
    main(task, task_cfg)
  File "/lustre1/home/erikde/master_thesis/GPTime-Series/GPTime/run.py", line 33, in main
    tasks[task](task_cfg)
  File "/lustre1/home/erikde/master_thesis/GPTime-Series/GPTime/model/train.py", line 292, in train
    torch.save(model.state_dict(), filename)
  File "/share/apps/software/PyTorch/1.3.1-fosscuda-2019b-Python-3.7.4/lib/python3.7/site-packages/torch/serialization.py", line 260, in save
    return _with_file_like(f, "wb", lambda f: _save(obj, f, pickle_module, pickle_protocol))
  File "/share/apps/software/PyTorch/1.3.1-fosscuda-2019b-Python-3.7.4/lib/python3.7/site-packages/torch/serialization.py", line 183, in _with_file_like
    f = open(f, mode)
FileNotFoundError: [Errno 2] No such file or directory: 'storage/experiments/test_folder_structure2/global-mase_loss-naive-256/global-mase_loss-naive-256.pt'
